src/index.js

```diff
@@ -61,6 +61,7 @@ export default function (options = {}, callback = function noop () {}) {
     }
 
     if (parsedOptions.protractorRetryConfig && retry) {
+      protractorArgs.splice(1, 1)
```

Comment:
Would it make sense to actually check what is being spliced and not just blindly assume it's at index 1 ?

Reply:
That would be better. But how can we make sure it's the protractor conf? Maybe a new option named *protractorConf* instead put the protractor config file in *protractorArgs*? Than we could search for it and then remove it.

Reply:
I think having something explicit here would be good; we could require that a `protractorConf` option is specified alongside `retryConfig`; this would mean a breaking change (and major version bump) but I'm okay with that if it makes this feature work. 

Users could still accidentally supply both the `protractorConf` option to flake _and_ a config file in the options passed to protractor after `--` but they'd run into the error in https://github.com/NickTomlin/protractor-flake/issues/93 so there would be _some_ indication that something is wrong.